Source/FakeItEasy.Tests/FakeOptionsBuilderTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace FakeItEasy.Tests
+{
+    using System;
+    using FakeItEasy.Creation;
+    using FluentAssertions;
+    using NUnit.Framework;
+    using TestHelpers;
+
+    public class FakeOptionsBuilderTests

```

Comment:
From [the docs](http://www.nunit.org/index.php/index.php?p=testFixture&r=2.6.3):
> Beginning with NUnit 2.5, the TestFixture attribute is optional for non-parameterized, non-generic fixtures.

And you can see it in [the CI server's records](http://teamcity.codebetter.com/viewLog.html?currentGroup=test&scope=%23teamcity%23FakeItEasy.Tests%23teamcity%23FakeOptionsBuilderTests&pager.currentPage=1&order=DURATION_DESC&recordsPerPage=20&filterText=FakeOptionsBuilder&status=&buildTypeId=bt929&buildId=208045&tab=testsInfo).

So I think we're safe without the tag. We've some classes with, some without. I think recently-added ones have omitted. If you're very concerned, we can publish a standard and update all classes one way or another. Then again, @mat-mcloughlin is currently converting _all_ the tests to XUnit, so the point may soon be moot.